packages/hub/util/bin-utils.js

```diff
@@ -67,6 +68,8 @@ HUB_ENVIRONMENT                   The environment the hub is running in. Possibl
 
   if (process.env.CI_SESSION_ID) {
     commander.ciSessionId = process.env.CI_SESSION_ID;
+  } else if (process.env.HUB_ENVIRONMENT === 'test') {
+    commander.ciSessionId = defaultTestingToken;
```

Comment:
Based on my previous comment, this should just go away.

Reply:
sure.

Reply:
@ef4 so one aspect about this i feel a little uneasy about is that we only have a CI session token if the hub is started with that as an env var when in production mode. the CI session token allows unimpeded access. I feel a little uncomfortable allowing the ability to have unimpeded access tot he hub in production mode when you did specifically set that up, that's why we're using an env var to drive this in production mode. In the test env, using a random value for this is totally fine. given that, I'm happy to make this random, but I dont think we should set this value when in production mode and there isn't a env var telling us that we are in CI.

Reply:
I see now, the way you have it is good as long as we randomize the generation.